packages/yoroi-extension/package.json

```diff
@@ -149,13 +149,13 @@
     "ws": "7.4.3"
   },
   "dependencies": {
-    "@cardano-foundation/ledgerjs-hw-app-cardano": "2.2.1",
+    "@cardano-foundation/ledgerjs-hw-app-cardano": "git+https://github.com/SebastienGllmt/ledgerjs-hw-app-cardano.git",
```

Comment:
We can switch to the main repo once https://github.com/cardano-foundation/ledgerjs-hw-app-cardano/pull/20 is deployed